command/main.go

```diff
@@ -220,6 +223,10 @@ func RunCustom(args []string, runOpts *RunOptions) int {
 		return 1
 	}
 
+	if runOpts.HCPTokenHelper == nil {
+		runOpts.HCPTokenHelper = &hcpvlib.InternalHCPTokenHelper{}
```

Comment:
It seems that the type in RunOptions is `hcpvlib.HCPTokenHelper` but here it is `hcpvlib.InternalHCPTokenHelper`. It is a bit confusing as to why that is the case? I have not looked into that library though.

Reply:
HCPTokenHelper is an interface